cmd/kubectl-gadget/deploy.go

```diff
@@ -34,6 +35,11 @@ func init() {
 		"traceloop", "",
 		true,
 		"enable the traceloop gadget")
+	deployCmd.PersistentFlags().StringVarP(
+		&runcHooks,
+		"runc-hooks", "",
+		"auto",
+		"how to attach runc hooks (auto, ldpreload)")
```

Comment:
I think `auto` could be confusing. `auto` could be interpreted like "you don't have to set it, we'll figure this out for you", but that's not the case currently. What about using "patched" for the case we are using edge with the patched runc version?

Reply:
Good point, if we choose patched and the user runs in a non Flatcar Edge system, it'll not work. I'm wondering if we can introduce an "auto" mode really quickly, i.e, it'll use "patched" if detects Flatcar Edge, otherwise will switch back to "ldpreload".